### DISCUSSION TITLE
what is only stage 2 stocks

### DISCUSSION THREAD

**Discussion by sjdhanasekaran** [pranjal-joshi/Screeni-py#133] [General]
what is  only stage 2 stocks.

**Comment by pranjal-joshi** [comment]
Hi @sjdhanasekaran

There are 4 theoretical stages that can belong to any stock.
Please read about the 4 stages here for further understanding and how/when to trade what stage: https://www.investopedia.com/articles/trading/08/stock-cycle-trend-price.asp
